messages/dev.generate.flag.md

```diff
@@ -158,60 +158,60 @@ Do you want to use the standard definition (character, summary, description, def
 
 # flagDescriptions.duration
 
-a unit and quantity of time (example: X minutes, 4 days). You can specify min/max/default.
+A unit and quantity of time, such as "10 minutes" or "4 days". You can optionally specify the minimum, maximum, or default values to take advantage of the built-in CLI validation rules.
 
 # flagDescriptions.option
 
-a choice from a defined list of string options. You can set the allowed values.
+Value is a single choice out of a pre-defined list of string values. You will then be asked for the list of allowed values.
```

Comment:
It'll ask you if the user can specify multiple values.  It's only a "single choice" if the user says not that prompt.

So I'd like to remove the word "single" ?

Reply:
Good call.